overlord/snapstate/snapstate_test.go

```diff
@@ -3753,9 +3753,6 @@ func (s *snapmgrTestSuite) TestEnsureRefreshRefusesLegacyWeekdaySchedules(c *C)
 	defer s.state.Unlock()
 	snapstate.CanAutoRefresh = func(*state.State) (bool, error) { return true, nil }
 
-	logbuf, restore := logger.MockLogger()
```

Comment:
Do we still check at some other place that the log message is generated into the log for invalid schedules?

Reply:
We don't log it anymore. Since the schedules are validated on "set", it shouldn't be possible to fail to parse something here due to an invalid config value

Reply:
That's a fair point, it makes sense for the auto-refresh code to be more resilient to failures than is typical. I'll change the error back to a log

Reply:
Moved back to logging and restored some tests